plugin/trino-redshift/src/test/java/io/trino/plugin/redshift/TestRedshiftCastPushdown.java

```diff
@@ -393,6 +433,33 @@ protected List<CastTestCase> supportedCastTypePushdown()
                 new CastTestCase("c_numeric_30_2", "integer", "c_integer"),
                 new CastTestCase("c_decimal_negative", "integer", "c_integer"),
 
+                new CastTestCase("c_char_50", "char(10)", "c_char_10"),
```

Comment:
Can we move all these new test cases after `new CastTestCase("c_decimal_negative", "bigint", "c_bigint")` so that integral cast cases come together?

Reply:
moved.